torch/onnx/_internal/exporter.py

```diff
@@ -410,7 +439,137 @@ def save(
             serializer.serialize(self, destination)
 
 
-class Exporter(abc.ABC):
+class FXGraphExtractor(abc.ABC):
+    """Abstract interface for FX graph extractor engines.
+    This class isolates FX extraction logic from the rest of the export logic.
+    That allows a single ONNX exporter that can leverage different FX graphs."""
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.input_adapter: InputAdapter = InputAdapter()
+        self.output_adapter: OutputAdapter = OutputAdapter()
+
+    @_beartype.beartype
+    def _export_fx_to_onnx(
+        self,
+        options: ResolvedExportOptions,
+        fx_module: torch.fx.GraphModule,
+        fx_module_args: Sequence[Any],
+    ) -> ExportOutput:
+        import torch.onnx._internal.fx.fx_exporter as fx_exporter
+        import torch.onnx._internal.fx.passes as passes
```

Comment:
Possible to move out?

Reply:
`_export_fx_to_onnx` weill be entirely remove in next design stages, there is no point in trying to fix it in this PR

Reply:
SG! I suggest a simple comment so we don't get confused in the interim.